cPP_DSC.adoc

```diff
@@ -993,34 +1026,43 @@ FCS_COP.1.1/SigVer:: The TSF shall perform _digital signature verification_ in a
 |List of Standards
 
 |RSA-PKCS
-|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#]
+|RSASSA-PKCS1-v1_5 using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224 SHA3-256, SHA3-384, SHA3-512#]
 |{empty}[*selection*: [.underline]#2048, 3072, 4096#] bits
 |{empty}[*selection*: [.underline]#RFC 8017, PKCS #1 v2.2 (Section 8.2); FIPS PUB 186-5 (Section 5.4)#] [RSASSA-PKCS1-v1_5]
 
-{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 202#] [SHA]
+{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 180-4, FIPS PUB 202#] [SHA]
 
 |RSA-PSS
-|RSASSA-PSS using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#]
+|RSASSA-PSS using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224 SHA3-256, SHA3-384, SHA3-512#]
 |{empty}[*selection*: [.underline]#2048, 3072, 4096#] bits
 |{empty}[*selection*: [.underline]#RFC 8017, PKCS#1v2.2 (Section 8.1); FIPS PUB 186-5 (Section 5.4)]# [RSASSA-PSS]
 
-{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 202#] [SHA]
+{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 180-4, FIPS PUB 202#] [SHA]
+
+|ECDSA
+|ECDSA on {empty}[*selection*: [.underline]#brainpoolP256r1, brainpoolP384r1, brainpoolP512r1, NIST P-256, NIST P-384, NIST P-521#] using {empty}[*selection*: [.underline]#SHA-224, SHA-256, SHA-384, SHA-512, SHA3-224 SHA3-256, SHA3-384, SHA3-512#]
+|{empty}[*selection*: [.underline]#256, 384, 512#] bits
+|{empty}[*selection*: [.underline]#ISO/IEC 14888-3:2018 (Sub Clause 6.6), FIPS PUB 186-5 (Sections 6.3.1, 6.4.2)]# [ECDSA]
 
-|ECDSA, Det-ECDSA
-|ECDSA on {empty}[*selection*: [.underline]#brainpoolP256r1, brainpoolP384r1, brainpoolP512r1, NIST P-256, NIST P-384, NIST P-521#] using {empty}[*selection*: [.underline]#SHA-256, SHA-384, SHA-512, SHA3-256, SHA3-384, SHA3-512#]
-|{empty}[*selection*: [.underline]#256, 384, 512, 521#] bits
-|{empty}[*selection*: [.underline]#ISO/IEC 14888-3:2018 (Sub Clause 6.6), FIPS PUB 186-5 (Section 6.4.2)]# [ECDSA]
+{empty}[*selection*: [.underline]#RFC 5639 (Section 3) [Brainpool Curves], NIST SP-800-186 (Section 3) [NIST Curves]#]
+
+{empty}[*selection*: [.underline]#ISO/IEC 10118-3:2018, FIPS PUB 180-4, FIPS PUB 202#] [SHA]
+
+|Det-ECDSA
```

Comment:
The table for FCS_COP.1/SigGen separates out ECDSA and Det-ECDSA, whereas this table merged them in one row. I don't know the rationale behind doing it, but I think it would be best if the tables were consistent. I don't have a preference either way (merged or separated).

Reply:
Resolved in #308.